etc/schema.json

```diff
@@ -5484,6 +5490,12 @@
                         "issuerdn": {
                             "type": "string"
                         },
+                        "subjectaltname": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
```

Comment:
nit: wouldn't it be a good time to add a description for the field, already?

Reply:
absolutely. Thank you!